README.md

```diff
@@ -1,82 +1,209 @@
+to rewrite as:
+     
+    try:
+        allele_number = [gnomad_exome.an for gnomad_exome in annotated_variant.gnomad_exomes][0]
+    except IndexError:
+        pass # no gnomad exomes annotation for the variant
+           
+#### Annotating a VCF using the client
+
+To annotate a VCF you can base your code on the VCFAnnotator object. This provides a basic implementation that
+will annotate a VCF file using a set of the available annotations. It uses [PyVCF](https://github.com/jamescasbon/PyVCF) to read and write to VCF files.
+
+    from varsome_api.vcf import VCFAnnotator
+    api_key = 'Your token'
+    vcf_annotator = VCFAnnotator(api_key=api_key, ref_genome='hg19', get_parameters={'add-all-data': 1, 'expand-pubmed-articles': 0})
+    vcf_file = 'input.vcf'
+    output_vcf_file = 'annotated.vcf'
+    vcf_annotator.annotate(vcf_file, output_vcf_file)
+        
+To annotate the VCF file with the annotations that you are interested in, you need only override 2 methods
+(`annotate_record` and `add_vcf_header_info`) in the VCFAnnotator class:
+
+    from varsome_api.vcf import VCFAnnotator
+    from vcf.parser import _Info
+    class MyVCFAnnotator(VCFAnnotator):
+    
+        def annotate_record(self, record, variant_result):
+            """
+            :param record: vcf record object
+            :param variant_result: AnnotatedVariant object
+            :return: annotated record object
+            """
+            record.INFO['gnomad_exomes_AN'] = variant_result.gnomad_exomes_an
+            # if you wish to also include the default annotations
+            # return super().annotate_record(record, variant_result)
+            return record
+        
+            
+        def add_vcf_header_info(self, vcf_template):
+            """
+            Adds vcf INFO headers for the annotated values provided
+            :param vcf_template: vcf reader object
+            :return:
+            """
+            vcf_template.infos['gnomad_exomes_AN'] = _Info('gnomad_exomes_AN', '.', 'Integer',
+                                                                 'GnomAD exomes allele number value', None, None)
+            # if you wish to also include the default headers
+            # super().add_vcf_header_info(vcf_template)
+                                                                 
+    api_key = 'Your token'
+    vcf_annotator = MyVCFAnnotator(api_key=api_key, ref_genome='hg19', get_parameters={'add-all-data': 1, 'expand-pubmed-articles': 0})
+    vcf_file = 'input.vcf'
+    output_vcf_file = 'annotated.vcf'
+    vcf_annotator.annotate(vcf_file, output_vcf_file)
+        
+        
+### API Documentation
+
+See [API documentation](https://api.varsome.com) for information on how to use the API and
+what values the API provides as a response to lookup requests.
+
+### How to get an API key
+
+You are generally not required to have an API key to use the API but, without one, the number of requests 
```

Comment:
Change this to: "You can use the API without an API key, but performance and the number of queries will be limited in order to safeguard our platform's reliability."